src/components/PatronBlock/PatronBlockForm.js

```diff
@@ -350,7 +350,12 @@ class PatronBlockForm extends React.Component {
 export default stripesFinalForm({
   initialValuesEqual: (a, b) => _.isEqual(a, b),
   navigationCheck: true,
-  subscription: { values: true },
+  subscription: {
+    pristine: true,
+    submitting: true,
```

Comment:
@Dmitriy-Litvinenko `submitting` and `pristine` are turned on by default so you don't need to list them here:

https://github.com/folio-org/stripes-final-form/blob/master/lib/StripesFinalFormWrapper.js#L97-L98

Reply:
Fixed